### DISCUSSION TITLE
Font icons not showing in terminal

### DISCUSSION THREAD

**Discussion by DevAlexSanz** [JanDeDobbeleer/oh-my-posh#3800] [Troubleshoot] [answered]
Hello, you are installing oh-my-posh to use it in my terminal, but it turns out that the icons do not appear, I have already installed several fonts (for example: CaskaydiaCove, FiraCode) and the problem persists

![image](https://user-images.githubusercontent.com/103162234/236638616-8b712c85-89dd-4408-b4b8-21d510cd08da.png)

**Comment by JanDeDobbeleer** [comment]
@DevAlexSanz this means you don't have a Nerd Font. See [here](https://ohmyposh.dev/docs/installation/fonts).

**Reply by DevAlexSanz** [reply]
Thanks for your answer, the final problem was, the new version of the font (3.0.0), apparently has a bug, I downloaded the font from the old version (2.3.3) and the problem was fixed ❤️

![image](https://user-images.githubusercontent.com/103162234/236641645-25fb005b-bd74-4ac0-ba04-6d6237de6b4d.png)

**Reply by JanDeDobbeleer** [reply, marked as answer]
@DevAlexSanz no, they migrated code points but I still need to make those the default. I'm going to try to do that this weekend. In the meantime you can also migrate the theme yourself. See [here](https://ohmyposh.dev/docs/faq#after-updating-my-nerd-font-to-a-newer-version-the-prompt-displays-unknown-characters).

**Reply by DevAlexSanz** [reply]
Thank you for your great contribution to the community!

**Reply by michalwidera** [reply]
Look's like 3.0.2 still have same issue.
Downgrade to 2.3.3 fixed same issue on mine case.

**Reply by DevAlexSanz** [reply]
> Look's like 3.0.2 still have same issue.
> Downgrade to 2.3.3 fixed same issue on mine case.

Yes, that's why the source was updated, I downloaded the first one that appeared when I went to GitHub 😅

**Reply by diegoesolorzano** [reply]
How can I downgrade or uninstall this?

**Reply by JanDeDobbeleer** [reply]
@diegoesolorzano download the font in the right version at nerd fonts on GitHub releases and install the included fonts.
